src/melee/ft/chara/ftZelda/ftzelda3.c

```diff
@@ -84,46 +79,144 @@ void ftZelda_SpecialAirN_StartAction(HSD_GObj* fighter_gobj)
     fp->cb.x21BC_callback_Accessory4 = &ftZelda_8013A8AC;
 }
 
-// 8013AA38 - 8013AACC (0x94 bytes)
-// https://decomp.me/scratch/XcH4E
+#ifdef MUST_MATCH
+
+#pragma push
+asm void ftZelda_8013AA38(HSD_GObj* fighter_gobj)
```

Comment:
Needed to revert a few functions in this file after correcting the type of `ftZeldaAttributes::x84`. It's probably not that hard to match if you want to try.